website/versioned_docs/version-v0.10.x/faq.md

```diff
@@ -99,7 +103,56 @@ If you are continuing to see this and the package repositories and vulnerability
 - update all packages without any scanner reports. This can be done by not providing a scanner report to Copa, and Copa will update all packages to the latest version available in the package repositories.
 
 ## Can I use Dependabot with Copa patched images?
+
 Yes, see [best practices](best-practices.md#dependabot) to learn more about using Dependabot with Copa patched images.
 
 ## Does Copa cause a buildup of patched layers on each patch?
+
 No. To prevent a buildup of layers, Copa discards the previous patch layer with each new patch. Each subsequent patch removes the earlier patch layer and creates a new one, which includes all patches applied since the original base image Copa started with. Essentially, Copa is creating a new layer with the latest patch, based on the base/original image. This new layer is a combination (or squash) of both the previous updates and the new updates requested. Discarding the patch layer also reduces the size of the resulting patched images in the future.
+
+## Why can't Copa patch my container image?
```

Comment:
I think a more descriptive title here for the specific error could be helpful? Also, can we add this to the current docs page https://github.com/project-copacetic/copacetic/blob/main/website/docs/faq.md

Reply:
How about something like: _Why am I getting 404 errors when trying to patch an image?_

I'll also get it added to the docs/faq.md :)